<launch>
  <arg name="detector_type" default="0" />
  <arg name="scenenet_true_scenenn_false" default="true" />
  <node
    pkg="line_ros_utility"
    type="histogram_line_lengths_node"
    name="histogram_line_lengths_node"
    args="$(arg detector_type)"
    output="screen"
 ></node>
  <node
    if="$(eval scenenet_true_scenenn_false == true)"
    pkg="line_ros_utility"
    type="scenenet_to_line_tools"
    name="scenenet_to_line_tools"
    output="screen"
 ></node>
  <node
    if="$(eval scenenet_true_scenenn_false == false)"
    pkg="line_ros_utility"
    type="scenenn_to_line_tools"
    name="scenenn_to_line_tools"
    output="screen"
 ></node>
  <node
    pkg="line_detection"
    type="line_extractor_node"
    name="line_extractor_node"
    output="screen"
 ></node>
  <node
    pkg="line_ros_utility"
    type="histogram_line_lengths_builder_node.py"
    name="histogram_line_lengths_builder_node"
    output="screen"
 ></node>

</launch>
